checks.d/docker_daemon.py

```diff
@@ -596,6 +597,8 @@ def _report_net_metrics(self, container, tags):
             # It is possible that the container got stopped between the API call and now
             self.warning("Failed to report IO metrics from file {0}. Exception: {1}".format(proc_net_file, e))
 
+        network.check_all(self, "docker.net", container['_proc_root'], tags=tags)
```

Comment:
I think there's also a good chunk of code that could be shared between `network.py` and `docker_daemon.py` for extracting metrics from /proc/net.